src/shared/components/search-filters/search-filters.component.html

```diff
@@ -17,11 +17,13 @@
     </select>
   </div>
 
-  <div id="dynamicFilters" *ngIf="isSearchableAndSearchInputsReady()">
-    <div class="form-group" *ngFor="let searchInput of searchInputs">
-      <ccd-field-write [caseField]="searchInput.field" [formGroup]="formGroup" [isExpanded]="true" (keyup.enter)="apply()"></ccd-field-write>
+  <ng-container ccdConditionalShowForm *ngIf="isSearchableAndSearchInputsReady()" [formGroup]="formGroup" [contextFields]="caseFields">
+    <div id="dynamicFilters">
+      <div class="form-group" *ngFor="let searchInput of searchInputs">
+        <ccd-field-write [caseField]="searchInput.field" [formGroup]="formGroup" [isExpanded]="true" (keyup.enter)="apply()"></ccd-field-write>
+      </div>
```

Comment:
Not sure it this can be unit tested, but I didn't see unit tests to cover this whole feature change.

Reply:
I did consider this long and hard however there are unit tests in the conditional-show-form-directive.spec.ts which test the show/hide conditions functionality.